### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by toensing**
Link for group table

### CODE REVIEW

**Review by MarcDjahan — APPROVED** [review]
Good protocol

File: README.md [hunk 1]
```diff
@@ -17,3 +17,7 @@ A *possible* workflow could look like:
 You can create as many branches as you need. When finished, plese clean up all you branches by deleting them **after** merged into the master branch. Only a ``master`` and a ``clean-*`` branch should be left.
 
 Github offers so called projects and (sub)-team, which if used can make any decision making within the group fairly transparent. 
+
+Please put your name in the table with the 4 tasks for the groups:
+https://docs.google.com/spreadsheets/d/1jOMLBlAiDwNceFCqvnptygjewFzQnyvgrPpci8o-LSU/edit?usp=sharing
+
```
Comment (JonasBorth) [inline README.md:23]: Nice

**Review by JonasBorth — APPROVED** [review]
Good link

**Comment by mmaelicke** [comment]
The pull request has been approved. You should be able to merge it yourself. If this does not work for you because of missing privileges, let me know by responding to this comment :)

File: README.md (hunk 1 above)
Comment (PatKac) [reply README.md:23]: good!

**Pull request merged**